dataparser/dataParser.py

```diff
@@ -0,0 +1,167 @@
+import os
```

Comment:
Please either remove V0 code from this branch or sync this branch with the already merged V0 code. Otherwise you will face merge issues when merging with master.